pyproject.toml

```diff
@@ -12,7 +12,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">= 3.9, < 4.0"
-pandas = ">= 2.0"
+pandas = "~2.1.4"
```

Comment:
@michaelpecan What's the point to fix this version? This is a python package, If the user has an issue with a specific version - He is able to fix it in his pyproject file